qa/suites/deepsea/tier4/upgrade/3-test-phase.yaml

```diff
@@ -26,6 +26,10 @@ tasks:
                         - 'zypper --non-interactive --no-gpg-checks rm gvfs-backends sle-sdk-release-POOL || true'
         - salt.command:
                 command: 'state.apply ceph.updates.salt'
+        - salt.command:
+                command: 'cmd.shell zypper -n in -f --from ses --force-resolution salt'
+        - salt.command:
+                command: 'cmd.shell rpm -qa | grep salt'
         - salt.command:
                 command: 'saltutil.sync_all'
```

Comment:
Hm, I wonder, would it make sense to let `salt.command` optionally take an array of strings? Then all four of the above invocations could be folded into a single invocation.

Reply:
Mhh, I don't mind keeping things explicit..